vm/interpreter/src/gas_tracker/price_list.rs

```diff
@@ -42,6 +43,52 @@ lazy_static! {
         hashing_base: 31355,
         compute_unsealed_sector_cid_base: 98647,
         verify_seal_base: 2000, // TODO revisit potential removal of this
```

Comment:
unrelated, but can this be removed now?

Reply:
I left it there because it is still a comment in lotus